### ISSUE TITLE
i386 is not supported.

### ISSUE THREAD

**Issue by Lolothepro** [makedeb/makedeb-docs#54]
https://docs.makedeb.org/installing/apt-repository/

```
curl -q 'https://proget.makedeb.org/debian-feeds/prebuilt-mpr.pub' | gpg --dearmor | sudo tee /usr/share/keyrings/prebuilt-mpr-archive-keyring.gpg 1> /dev/null
echo "deb [signed-by=/usr/share/keyrings/prebuilt-mpr-archive-keyring.gpg] https://proget.makedeb.org prebuilt-mpr $(lsb_release -cs)" | sudo tee /etc/apt/sources.list.d/prebuilt-mpr.list
sudo apt update
```
This would mean that Ubuntu would also have to search for i386 packages, while this is not supported by the repository.

**Comment by hwittenborn** [comment]
This doesn't cause any issues besides a little warning when running `apt update`, right? If so I think the flexibility to supporting more architectures in the repo if it were to ever be needed outweighs the downside of just seeing a little message. Though I don't know how GUI APT clients would handle this and if they'd show the message that seems like an error, in which case it might be a bit problematic.

makedeb is taking on some Rust code anyway (it's currently in the `alpha` branch but needs to be deployed still), which will probably require an architecture listing specific to the system anyway. So even if this is causing issues it should all be fixed in the near future

**Comment by hwittenborn** [comment]
This doesn't cause any issues besides a little warning when running `apt update`, right? If so I think the flexibility to supporting more architectures in the repo if it were to ever be needed outweighs the downside of just seeing a little message. Though I don't know how GUI APT clients would handle this and if they'd show the message that seems like an error, in which case it might be a bit problematic.

makedeb is taking on some Rust code anyway (it's currently in the `alpha` branch but needs to be deployed still), which will probably require an architecture listing specific to the system anyway (I'll have to check, but at the least i386 packages should be present to at least hide that specific issue). So even if this is causing issues it should all be fixed in the near future.

**Comment by Lolothepro** [comment]
Yes, it just adds a warning, but it is not possible to add && (to make several commands in one).
